deployments/platform-dev/docker-compose.yml

```diff
@@ -78,10 +78,12 @@ services:
     volumes:
       - db_data:/var/lib/mysql
     environment:
-      - MYSQL_ROOT_PASSWORD
+      - MYSQL_ROOT_PASSWORD=uns3cur3
```

Comment:
this file is not meant to be used by developers but for deployments

on deployment servers, we always use .env files to configure unique env vars

please revert back this file, but feel free to update/create docker-compose files outside of deployments/ directory to make the maintainers' life easier

Reply:
okay sure